src/lib.rs

```diff
@@ -20,6 +20,7 @@ pub use syscalls::Syscall;
 #[derive(Debug, Clone, Copy)]
 pub enum Action {
     Kill,
```

Comment:
```suggestion
    KillProcess,
```
Sorry for changing my mind on this, but can we rename this one to `KillProcess` instead?